components/script/dom/window.rs

```diff
@@ -716,6 +723,33 @@ impl<'a> WindowHelpers for JSRef<'a, Window> {
         self.timers.suspend();
     }
 
+    fn debug_reflow_events(self, goal: &ReflowGoal, query_type: &ReflowQueryType, reason: &ReflowReason) {
```

Comment:
This function is only used by reflow(). Wouldn't make sense to keep it private and within the scope of the user class/struct?